x/project/types/msgs.go

```diff
@@ -52,9 +54,8 @@ func NewMsgCreateProject(senderDid didexported.Did, projectData json.RawMessage,
 
 func (msg MsgCreateProject) ToStdSignMsg(fee int64) legacytx.StdSignMsg {
 	accNum, accSeq := uint64(0), uint64(0)
-	stdFee := legacytx.NewStdFee(0, sdk.NewCoins(sdk.NewCoin(
-		ixotypes.IxoNativeToken, sdk.NewInt(fee))))
-	memo := viper.GetString(flags.FlagMemo)
+	stdFee := legacytx.NewStdFee(0, sdk.NewCoins(sdk.NewCoin(ixotypes.IxoNativeToken, sdk.NewInt(fee))))
+	memo := viper.GetString(flagMemo)
```

Comment:
@ig-shaun @demondayza @steveixo 
Relates to
```
// TODO: This was removed from the CosmosSDK without clear reason. However, this does not seem important, consider removing this entirely.
flagMemo = "memo"
```

Cosidering this as a good alternative, unless someone says differently.

Reply:
I feel you can remove this.

Reply:
It seemed to be hardcoded anyway if im reading right.